amd64-hassio-addon/run.sh

```diff
@@ -10,22 +10,45 @@ HASSIO_DEBUG_OTAU="$(jq --raw-output '.debug_otau' $CONFIG_PATH)"
 HASSIO_DECONZ_WEB_PORT="$(jq --raw-output '.web_port' $CONFIG_PATH)"
 HASSIO_DECONZ_WS_PORT="$(jq --raw-output '.websockets_port' $CONFIG_PATH)"
 HASSIO_DECONZ_DEVICE="$(jq --raw-output '.deconz_device' $CONFIG_PATH)"
+HASSIO_DECONZ_VNC_MODE=":$(jq --raw-output '.vnc.active' $CONFIG_PATH)"
+HASSIO_DECONZ_VNC_DISPLAY=":$(jq --raw-output '.vnc.port - 5900' $CONFIG_PATH)"
+HASSIO_DECONZ_VNC_PORT="$(jq --raw-output '.vnc.port' $CONFIG_PATH)"
+HASSIO_DECONZ_VNC_PASSWORD="$(jq --raw-output '.vnc.password' $CONFIG_PATH)"
+
+DECONZ_OPTS="--auto-connect=1 \
+        --dbg-info=$DEBUG_INFO \
+        --dbg-aps=$DEBUG_APS \
+        --dbg-zcl=$DEBUG_ZCL \
+        --dbg-zdp=$DEBUG_ZDP \
+        --dbg-otau=$DEBUG_OTAU \
+        --http-port=$DECONZ_WEB_PORT \
+        --ws-port=$DECONZ_WS_PORT"
 
 echo "[Hass.io] Starting deCONZ Hass.io Addon..."
 echo "[Hass.io] Current deCONZ version: $DECONZ_VERSION"
 echo "[Hass.io] Web UI port: $HASSIO_DECONZ_WEB_PORT"
 echo "[Hass.io] Websockets port: $HASSIO_DECONZ_WS_PORT"
 echo "[Hass.io] deCONZ device: $HASSIO_DECONZ_DEVICE"
 
-/usr/bin/deCONZ \
-    -platform minimal \
-    --auto-connect=1 \
-    --dbg-info=$HASSIO_DEBUG_INFO \
-    --dbg-aps=$HASSIO_DEBUG_APS \
-    --dbg-zcl=$HASSIO_DEBUG_ZCL \
-    --dbg-zdp=$HASSIO_DEBUG_ZDP \
-    --dbg-otau=$HASSIO_DEBUG_OTAU \
-    --http-port=$HASSIO_DECONZ_WEB_PORT \
-    --ws-port=$HASSIO_DECONZ_WS_PORT \
-    --dev=$HASSIO_DECONZ_DEVICE
- 
\ No newline at end of file
+if [ "$HASSIO_DECONZ_VNC_MODE" == "true" ]; then
+  echo "[Hass.io] Starting VNC Server on Port $HASSIO_DECONZ_VNC_PORT"
```

Comment:
Architecturally, I think we should just always enable the VNC mode (are there drawbacks to doing this?). Thoughts?

Reply:
Capacity (mem and cpu) on Raspberry pis´s perhaps ... ?

Reply:
deconz will probably use more resources with GUI and we open another port which must be protected with a separate password. Not everyone needs it, so it think it should be disabled by default.